analytics_data_api/v0/connections.py

```diff
@@ -0,0 +1,69 @@
+import json
+import time
+
+from boto.connection import AWSAuthConnection
+from django.conf import settings
+from elasticsearch import Connection
+
+
+class BotoHttpConnection(Connection):
+    """
+    Uses AWS configured connection to sign requests before they're sent to
+    elasticsearch nodes.
+    """
+
+    connection = None
+
+    def __init__(self, host='localhost', port=443, **kwargs):
+        super(BotoHttpConnection, self).__init__(host=host, port=port, **kwargs)
+        connection_params = {'host': host, 'port': port}
+
+        # Set the credentials through settings.  If not provided, boto will attempt to use
+        # default environment variables.
```

Comment:
We don't instantiate the connection.  Rather, the class is passed into elasticsearch-dsl's connection pool, thus changing the modifying the constructor won't get us what we want without modifications to elasticsearch-dsl.

The fallback if the settings aren't provided is that the environment variables are used.  @mulby -- does that work for the pipeline?  I suppose we'll need to conditionally import `django.conf.settings`.

Another way to do this might be to set the environment variable as defined in the django settings.  That feels weird though.  I'm open to other ideas!